src/core/dfn.js

```diff
@@ -32,3 +46,27 @@ export function run(conf) {
       }, $dfn);
   });
 }
+
+function autoPluralizeDfns() {
+  const links = [...document.querySelectorAll("a")];
+  const dfns = [...document.querySelectorAll("dfn")];
+
+  return function shouldPluralize(dfn) {
+    const plural = pluralize(norm(dfn.textContent));
+    if (
+      // has overridden to not do pluralization
+      dfn.hasAttribute("data-lt-noPlural") ||
+      // no <a> references the plural term
+      !links.find(({ textContent }) => norm(textContent) === plural) ||
+      // there is any dfn with plural term in data-lt or textContent
+      dfns.find(
+        ({ textContent, dataset }) =>
+          norm(textContent) === plural ||
```

Comment:
is there some way to reuse this check from line 60 here? `norm(textContent) === plural` is being done twice.